app/forms/provider_interface/interview_wizard.rb

```diff
@@ -39,7 +42,7 @@ def date
     end
 
     def date_and_time
-      Time.zone.local(date.year, date.month, date.day, parsed_time.hour, parsed_time.min) if date.is_a?(Date)
+      Time.zone.local(date.year, date.month, date.day, parsed_time.hour, parsed_time.min) if date.is_a?(Date) && parsed_time.is_a?(Time)
```

Comment:
If I enter a time as `6767` I am getting the following error: `argument out of range` when it attempts to parse the time. Maybe we should also ensure that the time is within a parseable range?

Reply:
This should be addressed now – we now check that the time is in the right format before trying to parse it